content/applications/general/users/access_rights.rst

```diff
@@ -52,6 +52,65 @@ The :guilabel:`Administration` field in the :guilabel:`Access Rights` tab has th
+   administrator who does not need to track time.
+
+To manage specific permissions, :ref:`developer mode <developer-mode>` must be enabled. After that,
+navigate to the :menuselection:`Settings` app. Then click :guilabel:`Manage Users`, select a user,
+and go to the :guilabel:`Technical Access Rights` tab. From here, :guilabel:`Groups` can be edited,
+and specific access rights can be managed across the various sections. If no changes are made to
+these *Groups*, then their permissions will mirror the selections made in the :guilabel:`Access
+Rights` tab.
+
+- :guilabel:`Selected groups`: a list of detailed access rights, set by choices made in the
+  :guilabel:`Access Rights` tab.
+- :guilabel:`Groups added automatically`: *implied* permissions that are *inherited* with the
+  explicit permissions already granted to the user. The values here will match the values listed
+  under a given *Group*'s form located under the :menuselection:` Users & Companies --> Groups`
+  menu, in the :guilabel:`Inherited` tab.
+
+.. image:: access_rights/tech-access-rights.png
+   :alt: The technical access rights tab opened up for a user profile.
+
+.. example::
+   When the *Sales Administrator* permission set is assigned to a user, then the *Canned Responses
+   Administrator* permissions are inherited automatically. These assignments are reflected across
+   the values listed in the :guilabel:`Selected Groups` and :guilabel:`Groups added automatically`
+   tables, respectively.
+
+To add a permission to this user profile, click :guilabel:`Add a line` in the :guilabel:`Selected
+groups` table, and then add permissions to this user profile. To remove a permission, click the
+:icon:`fa-times` :guilabel:`(cancel)` at the end of that permission's row.
+
+.. warning::
+   Removing permissions from the *Selected groups* list can impact what permissions are listed in
+   the *Groups added automatically* list, since selected permission groups inform what permission
+   groups are added automatically.
+
+Clicking on the permission itself will open a *group management form*. Learn more about
+:ref:`managing groups <access-rights/groups>`.
+
+Any permission in the :guilabel:`Groups added automatically` section are implied or required by the
+permission shown in the :guilabel:`Selected groups` section. These cannot be removed, but more users
+can be given these permissions by clicking on the permission itself, and then adding the user to
+that permission's group.
+
+.. note::
+   Any permission in green is already provided by another permission (for example, setting the
+   **Website** permission to :guilabel:`Editor and Designer` will also give that user the
+   :guilabel:`Restricted Editor` permission). Any permissions in red are conflicting and cannot be
+   active at the same time. Any permissions in *italics* is implied by a :guilabel:`Selected group`
+   (these are usually found in the :guilabel:`Groups added automatically`).
```

Comment:
Great detail! Again totally optional, but may be visually helpful to format these in a bullet list to appeal to readers skimming through the doc :)
```suggestion
   - Any permission in green is already provided by another permission (for example, setting the
     :guilabel:`Website` app's permission to :guilabel:`Editor and Designer` will also give that
     user the :guilabel:`Restricted Editor` permission).
   - Any permissions in red are conflicting and cannot be active at the same time.
   - Any permissions in *italics* is implied by a :guilabel:`Selected group` (these are usually
     found in the :guilabel:`Groups added automatically`).
```